src/views/CampusSafety/components/Header/Header.module.scss

```diff
@@ -27,19 +27,56 @@
   position: sticky;
   top: -1rem;
   margin-bottom: 1rem;
-  z-index: 1; // this component defaults to a higher z-index that overlays quick search
+  z-index: 10; // this component defaults to a higher z-index that overlays quick search
```

Comment:
I'm just curious, why did the z-index value need to be changed?

Reply:
Changing that prevents the FirstName and Lastname from showing when you scroll, like it was in Bens initial comment/screenshot

Reply:
Gotcha, makes sense 👍